crates/server/src/authority/catalog.rs

```diff
@@ -20,8 +20,9 @@ use crate::proto::rr::{
 };
 use crate::{
     authority::{
-        AuthLookup, AuthorityObject, EmptyLookup, LookupError, LookupObject, LookupOptions,
-        MessageResponse, MessageResponseBuilder, ZoneType,
+        AuthLookup, AuthorityObject, EmptyLookup, LookupControlFlow, LookupControlFlow::*,
```

Comment:
Given the downsides of `*` imports, would prefer to keep this as a local import, used only just before a `match` that needs it.